### PR TITLE
SPU: Fix "SPU Accurate Reservations" off

### PR DESCRIPTION
**Pull request by elad335**
Make it compatible with many more games.

### CODE REVIEW

File: rpcs3/Emu/Memory/vm.cpp [hunk 1]
```diff
@@ -112,15 +119,15 @@ namespace vm
 					reservation_notifier(addr).notify_all();
 				}
 
+				if (cpu && !had_wait && cpu->test_stopped())
+				{
+					//
```
Comment (MSuih) [inline rpcs3/Emu/Memory/vm.cpp:124]: ?

Comment (elad335) [reply rpcs3/Emu/Memory/vm.cpp:124]: Prevents warning of test_stopped() (discard return value)

Comment (MSuih) [reply rpcs3/Emu/Memory/vm.cpp:124]: Ah, okay. Could've written that explanation in the comment

**Comment by digant73** [comment]
I confirm that with this PR all my games properly start when **SPU Accurate Reservations** is set to "false" in their custom configuration files although I didn't see performance improvements.
I noted that only killzone 3 needed a clear of all the caches, otherwise the game was started (game menu was loaded) but then the selected game chapter was not started, if it can suggest you something to check/fix in the PR

**Pull request merged**